zammad/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: zammad
-version: 10.1.1
-appVersion: 6.1.0-24
+version: '11.0.0'
+appVersion: '6.2'
```

i would prefer to use it like this, to prevent unwanted updates, even if its only patchlevel.

```suggestion
appVersion: 6.2.0
```

---

That's precisely what we intend to change here (prepared by https://github.com/zammad/zammad/pull/4920). We can choose between two evils here:

1. Don't update automatically. This means that docker and k8s installations stay on outdated patchlevel versions of Zammad and don't receive updates unless somebody submits a PR.
2. Update within patchlevels. This causes automatic updates, but only within patch level updates.

We think that the latter option is the better one, as it is a good compromise between stability and being up-to-date without maintenance burden. Having a software with bugs that are in reality already solved is not what admins should want.

Can you elaborate your reasons against this change, please?

---

This is only about changing the *default* behaviour. It's still possible to use a fixed version, or even a very loose one like `:stable` or `:latest`.

---

Usually you don't want to use flloating tags in production environments for sevaral reasons. Most important ones are unpredictability of image content and the risk of breaking changes. 

Even if zammad version is only a patch lavel change the whole docker container is rebuild so also changes in the os can happen.

So i see it exactly the other way around. 
If you want to use a flaotaing tag it should be an opt in.
Never the default.

---

We follow your advice, @monotek